### DISCUSSION TITLE
Continuous dashboard refresh

### DISCUSSION THREAD

**Discussion by frogg1ven** [gapitio/gapit-htmlgraphics-panel#181] [Q&A]
I have written a date selection tool in Grafana, where you can change the week, day, and shift using several buttons. When a button is clicked, the following action is performed:

```javascript
htmlGraphics.locationService.push("dashboard?refresh=5s&from=" + dateFrom + "&to=" + dateTo);
```
You can notice that I have also added "refresh=5s" to the links so that the dashboard is refreshed every 5 seconds. However, it doesn't work as I expected. Every other click on one of the buttons causes the "refresh=5s" parameter to disappear from the link, and the dashboard doesn't refresh. Is there any way for the panels to refresh their data at selected time intervals and make it work consistently?

**Comment by ZuperZee** [comment]
Weird. I'm also encountering this.
